client/src/themes/theme.js

```diff
@@ -3,7 +3,7 @@ import { createMuiTheme } from '@material-ui/core';
 
 export const theme = createMuiTheme({
   typography: {
-    fontFamily: '"Roboto"',
+    fontFamily: '"Open Sans", sans-serif',
   },
   primary: '#f04040',
   secondary: '#1f1f1f',
```

Comment:
Can you change the colors in the theme.js as well, so when a TextField is highlighted, it uses our spec's green colors?

Reply:
Will work on this